doc/sidewalk_known_issues.rst

```diff
@@ -8,37 +8,44 @@ See the list of known issues that are valid for the current state of development
 v1.14.5-dev1
 ------------
 
-* KRKNWK-17374: Zephyr fatal error (assertion in semaphore module) after link down on FSK.
-    The issue occurs in Sidewalk 1.14 libraries, will be fixed in 1.16.
-    It reproduces rarely - once per few days.
-    In release mode device resets automatically, in debug mode it need to be reset manually.
+* KRKNWK-17374: Zephyr fatal error after enabling downlink on FSK
```

Comment:
Rephrased this part a bit, please check

Reply:
"link down" means disconnection between nRF device and gateway, "downlink" means a message form gateway (AWS) to nRF device. Please consider:

```suggestion
* KRKNWK-17374: Sporadic Zephyr fatal error after disconnecting on FSK
```